src/mse.ts

```diff
@@ -188,43 +179,49 @@ export class MSERep extends EventEmitter implements MSE {
 
 	// Rundown basics task
 	async createRundown(profileName: string, playlistID?: string, description?: string): Promise<VRundown> {
-		let playlistExists = false
-		const date = new Date()
-		description = description ? description : `Sofie Rundown ${date.toISOString()}`
+		await this.assertProfileExists(profileName)
+
+		description = description ? description : `Sofie Rundown ${new Date().toISOString()}`
+		playlistID = playlistID ? playlistID.toUpperCase() : uuid.v4().toUpperCase()
```

Comment:
Depending on whether we get empty strings, this can be simplified to:
```suggestion
		playlistID = (playlistID ?? uuid.v4()).toUpperCase()
```

Reply:
I believe that we do get empty strings here